api/dp-openapi.yaml

```diff
@@ -591,6 +591,350 @@ paths:
+    post:
+      tags:
+      - project
+      description: make page available for proofreading, change state to temp if necessary
+      parameters:
+      - name: roundid
+        in: path
+        description: The round identifier (as defined by array_keys($Round_for_round_id_))
+        required: true
+        schema:
+          type: string
+      - name: projectid
+        in: path
+        description: The project id
+        required: true
+        schema:
+          type: string
+      - name: page
+        in: path
+        description: the name of the page, must be in temp state of saved state and owned by user
+        required: true
+        schema:
+          type: string
+      responses:
+        200:
+          description: Page data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/proof_page'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        404:
+          $ref: '#/components/responses/NotFound'
+        429:
+          $ref: '#/components/responses/RateLimitExceeded'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+
+  /wordcheckdata:
+    get:
+      tags:
+      - project
+      description: data needed if wordcheck is run
+      responses:
+        200:
+          description: data returned correctly
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/wordcheck_data'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        404:
+          $ref: '#/components/responses/NotFound'
+        422:
+          $ref: '#/components/responses/InvalidCharacters'
+        429:
+          $ref: '#/components/responses/RateLimitExceeded'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+
+  /projects/{projectid}/wordcheck:
+    put:
+      tags:
+      - project
+      description: checks text for bad words
+      parameters:
+      - name: projectid
+        in: path
+        description: ID of project
+        required: true
+        schema:
+          type: string
+      requestBody:
+        description: text to check
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/page_check'
+      responses:
+        200:
+          description: description of checked text
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/checked_text'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        404:
+          $ref: '#/components/responses/NotFound'
+        422:
+          $ref: '#/components/responses/InvalidCharacters'
+        429:
+          $ref: '#/components/responses/RateLimitExceeded'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+
+  /rounds/{roundid}/projects/{projectid}/pages/{page}/suggestions:
```

Comment:
This should be after the one for `/rounds/{roundid}/projects/{projectid}/pages/{page}` in the file.